odoo/addons/base/models/assetsbundle.py

```diff
@@ -283,6 +283,7 @@ def get_attachments(self, extension, ignore_version=False):
                                 => web/assets/%-self.version/name.extension.
         """
         unique = "%" if ignore_version else self.version
+        extra = '' if unique else extra
```

Comment:
Ho this was a mystake to commit that, good catch

This is actually not correct in the current version but the idea would be that extra should be enought to fully define the bundle. This is actually not the case because of rtl. The the website id part of the extra is not really usefull and we could match an existing attachment if it has the same unique.